MonoGame.Framework/MonoGame.Framework.AndroidCore.csproj

```diff
@@ -72,7 +72,7 @@ This package provides you with MonoGame Framework that works on Android.</Descri
   </ItemGroup>
 
   <ItemGroup>
-    <Content Include="MonoGame.Framework.AndroidCore.targets" PackagePath="build" />
```

Comment:
I need an extra modification here, once done with my current changes I'll add it.